Rubberduck.Refactorings/ExtractInterface/ExtractInterfaceModel.cs

```diff
@@ -47,14 +50,15 @@ public ExtractInterfaceModel(RubberduckParserState state, QualifiedSelection sel
 
             InterfaceName = $"I{TargetDeclaration.IdentifierName}";
 
-            Members = declarations.Where(item => item.ProjectId == TargetDeclaration.ProjectId
-                                                 && item.ComponentName == TargetDeclaration.ComponentName
-                                                 && (item.Accessibility == Accessibility.Public || item.Accessibility == Accessibility.Implicit)
-                                                 && MemberTypes.Contains(item.DeclarationType))
-                                   .OrderBy(o => o.Selection.StartLine)
-                                   .ThenBy(t => t.Selection.StartColumn)
-                                   .Select(d => new InterfaceMember(d))
-                                   .ToList();
+            Members = new ObservableCollection<InterfaceMember>(declarations.Where(item =>
```

Comment:
`ObservableCollection` shouldn't be static.  Doesn't that leak everything it's bound to?